schemas/cloudflare/account-1.yml

```diff
@@ -30,6 +30,8 @@ properties:
       - email
   apiCredentials:
     "$ref": "/common-1.json#/definitions/vaultSecret"
+  enforceTwofactor:
```

Comment:
NIT: isn't it `enforceTwoFactor` ?

Reply:
I converted the [terraform provider parameter](https://registry.terraform.io/providers/cloudflare/cloudflare/latest/docs/resources/account) from snake_case to camelCase to match the style of the other parameters

Often times we keep the snake_case version of terraform parameters when we use them in terraform resources schemas. Here it's not really a resource definition for more of an account property that gets turned into a terraform parameter under the hood.

I don't really have a good argument either way. I think it's fine to merge as-is?